packages/prepo-utils/src/getRpcOverrides.ts

```diff
@@ -0,0 +1,37 @@
+import { parse } from 'query-string'
+import fromPairs from 'lodash/fromPairs'
+
+/*
+ * Only load trusted RPCs via URL. Otherwise, a user may be referred to prePO
+ * via a link containing a malicious URL.
+ */
+const isRpcAllowed = (rpcUrl: string) => new URL(rpcUrl).host === 'rpc.vnet.tenderly.co'
+
+export const getRpcOverrides = (): Record<number, string | undefined> => {
+  if (typeof window === 'undefined') return {}
+
+  const query = parse(window.location.search)
+
+  let rawRpcs: string[]
+
+  if (typeof query.rpc === 'string') {
+    rawRpcs = [query.rpc]
+  } else if (Array.isArray(query.rpc)) {
+    rawRpcs = query.rpc.filter((str): str is string => typeof str === 'string')
+  } else {
+    rawRpcs = []
+  }
```

Comment:
I don't think we need these lines?
```suggestion
  }
```

Reply:
We don't if we initialize `rawRpcs = []` which I didn't do before 👍